src/engine/enginechannel.h

```diff
@@ -39,7 +39,8 @@ class EngineChannel : public EngineObject {
     };
 
     EngineChannel(const ChannelHandleAndGroup& handle_group,
-                  ChannelOrientation defaultOrientation = CENTER);
+                  ChannelOrientation defaultOrientation = CENTER,
+                  bool isInput = false);
```

Comment:
I was ambivalent about naming this because I was thinking that auxiliary inputs should also have latency compensation. But now I am questioning that because auxiliary inputs may be used for connecting external software. It is not clear why we have two different types of inputs with different capabilities and it is a burden to maintain them both.